images/simple-image.mk

```diff
@@ -37,7 +37,7 @@ IMG = $(REGISTRY)/$(PROJECT)/test-infra/$(IMAGE_NAME)
 TAG := $(shell date +v%Y%m%d)-$(shell git describe --always --dirty --match '^$$')
 
 build:
-	docker build --no-cache --pull -t $(IMG):$(TAG) -f $(DOCKERFILE) $(DOCKERBUILDARGS) ../..
```

Comment:
IIRC, `--no-cache` was added as we want to always update `gcloud` when we build the `prow-tests` image. Can we make an exception to keep this option when building the `prow-tests` image?

Reply:
I see. --no-cache doesn't hurt images other than `prow-tests` as far as I can tell. Let me add it back